doctr/datasets/loaders/detection/core.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (C) 2021, Mindee.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+import os
+import json
+import cv2
+import tensorflow as tf
+import numpy as np
+from typing import List, Tuple
+
+
+def load_annotation(
+    labels_path: str,
+    img_name: str,
+) -> Tuple[List[List[List[int]]], List[List[List[int]]]]:
+    """Loads detection annotations (boxes) for an image, from a folder containing
+    annotations in json files for each image
+
+    Agrs:
+        labels_path: path to the folder containing all json annotations
+        img_name: name of the image to find the corresponding json annotation
+
+    Returns:
+        A tuple of 2 lists: a list of polygons to draw on the segmentation gt map,
+        and a list of ambiguous polygons to mask during the training
```

Comment:
I think it might be better, to keep all boxes together, but having a boolean vector saying whether the box is ambiguous or not (it will be easier to batch, and thus easier to avoid taking it into account in the loss later on)